tldw_Server_API/app/api/v1/endpoints/admin/admin_personalization.py

```
@@ -0,0 +1,29 @@
+
+router = APIRouter()
+
+
+@router.post(
+    "/personalization/consolidate",
+    response_model=dict,
+    dependencies=[Depends(check_rate_limit)],
+)
+async def trigger_personalization_consolidation(
+    user_id: str | None = Query(None, description="User ID to consolidate; defaults to single-user id"),
+    principal: AuthPrincipal = Depends(get_auth_principal),
```

Ruff (0.14.14)

[warning] Unused function argument: `principal`

(ARG001)